kube/services/jobs/psql-db-backup-encrypt-job.yaml

```diff
@@ -38,6 +38,11 @@ spec:
                   key: environment
             - name: JENKINS_HOME
               value: "devterm"
+            - name: slackWebHook
+              valueFrom:
+                configMapKeyRef:
+                  name: global
+                  key: slack_alarm_webhook
```

Comment:
Can we add optional: true like here:

https://github.com/uc-cdis/cloud-automation/blob/ef9f921d93b49dafb1005ff8eda1226bd21fede4/kube/services/jobs/etl-job.yaml#L73-L78

Reply:
added